### ISSUE TITLE
System screen is blank, but everything else works fine

### ISSUE THREAD

**Issue by jshrek** [freescout-helpdesk/freescout#3042]
Freescout is working fine as far as I can tell, but I decided to see if there were any updates, and when I click on Manage > System I get a blank screen (on both Firefox and Chrome). All other pages are fine, just the System page is blank.

I updated Freescout a couple of months ago, but the System screen was working then.
I have not done any kind of update since then.
I can not remember excatly which version I updated too, but again it was only a couple months ago so it is not too far out of date.

There is an error file in the /public_html/public folder with this line in it:
[23-May-2023 09:35:28 America/Chicago] PHP Fatal error:  Allowed memory size of 536870912 bytes exhausted (tried to allocate 32768 bytes) in /home/mydomain/public_html/overrides/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php on line 115

Seems to be allocating a LOT of memory?? Any ideas on whats causing this?

**Comment by freescout-helpdesk** [comment]
We need a full error message from Manage > Logs > App Logs

**Comment by jshrek** [comment]
I went to system page again (which was blank) and then went to App Logs ...

The queue-jobs.log is empty.

And the fetch-emails.log just has the fetching...
[2023-05-23 12:47:03] Fetching UNREAD emails for the last 3 days.
[2023-05-23 12:47:03] Mailbox: VOH box11
[2023-05-23 12:47:03] Folder: INBOX
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Archive
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Drafts
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Trash
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Junk
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.spam
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Sent
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Mailbox: VOH box10
[2023-05-23 12:47:03] Folder: INBOX
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Archive
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Drafts
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Trash
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Junk
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.spam
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Sent
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Mailbox: VOH box9
[2023-05-23 12:47:03] Folder: INBOX
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Drafts
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Trash
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Junk
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.spam
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Sent
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Archive
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Mailbox: VOH box8
[2023-05-23 12:47:03] Folder: INBOX
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.spam
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Archive
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Trash
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Sent
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Junk
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Drafts
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Mailbox: VOH box7
[2023-05-23 12:47:03] Folder: INBOX
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Archive
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.spam
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Trash
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Sent
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Junk
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:03] Folder: INBOX.Drafts
[2023-05-23 12:47:03] Fetched: 0
[2023-05-23 12:47:04] Mailbox: VOH box6
[2023-05-23 12:47:04] Folder: INBOX
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Drafts
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Trash
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Junk
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.spam
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Sent
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Archive
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Mailbox: VOH box5
[2023-05-23 12:47:04] Folder: INBOX
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Drafts
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Trash
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Junk
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.spam
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Sent
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Archive
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Mailbox: VOH box4
[2023-05-23 12:47:04] Folder: INBOX
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Drafts
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Trash
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Junk
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.spam
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Sent
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Archive
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Mailbox: VOH box3
[2023-05-23 12:47:04] Folder: INBOX
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Drafts
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Trash
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Junk
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.spam
[2023-05-23 12:47:04] Fetched: 0
[2023-05-23 12:47:04] Folder: INBOX.Sent
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.Archive
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Mailbox: VOH box2
[2023-05-23 12:47:05] Folder: INBOX
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.Drafts
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.Trash
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.Junk
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.spam
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.Sent
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.Archive
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Mailbox: VOH box1
[2023-05-23 12:47:05] Folder: INBOX
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.Drafts
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.Trash
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.Junk
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.spam
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.Sent
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Folder: INBOX.Archive
[2023-05-23 12:47:05] Fetched: 0
[2023-05-23 12:47:05] Fetching finished

**Comment by freescout-helpdesk** [comment]
Re-update your FreeScout using manual updating method: https://github.com/freescout-helpdesk/freescout/wiki/Updating-FreeScout

**Comment by jshrek** [comment]
ok I did manual update and System screen is working again.

**Comment by jshrek** [comment]
So I noticed my /system/status screen was blank again.

I have manually updated to v1.8.83 and migrated db and cleared cache, but the  page is still blank.
Everything else appears to be functioning fine.

I am then seeing the following error in the LOG:
![log](https://github.com/freescout-helpdesk/freescout/assets/19673842/436a1874-2d32-46d7-b5c4-25e5c66027f9)

It seems like memory allocation should be more than enough, but something is not quite right!
What else should I check?
